src/utils/path_helpers.py

```diff
@@ -57,17 +75,19 @@ def create_staging_config(config: dict) -> dict:
     paths = get_paths(config)
     berd_path = paths["berd_path"]
 
+    survey_year = str(config["years"]["survey_year"])    
+    msg = ""
```

Comment:
I think you can get rid of these 2 lines... but are you actually calling your new function?